taskfiles/k8s.yml

```diff
@@ -239,3 +239,16 @@ tasks:
     cmds:
       - 'echo "~~~ Creating operator v2 helm e2e tests :k8s:"'
       - ./hack/v2-helm-setup.sh
+
+  package-and-publish-operator-chart:
+    cmds:
+      - defer: 'helm registry logout registry-1.docker.io'
+      - helm registry login registry-1.docker.io -u {{.DOCKERHUB_USER}} --password {{.DOCKERHUB_TOKEN}}
+      - mkdir helm-operator-release
```

Comment:
nit: I think it'd be better to do this in a tmp directory via `export PKG_DIR=$(mktemp -d)` and to copy the source files into the temp directory rather than changing them in the repository. Doing so will make the release process more hermetic and remove any chance of old/stale builds impacting current releases.

Reply:
Ok